### ISSUE TITLE
Unable to deploy a new container when an existing app container is running.

### ISSUE THREAD

**Issue by dmitrinesterenko** [dmitrinesterenko/blog#70]
```sh
docker ps -q -f "name=blog" | xargs docker stop | xargs docker rm
"docker stop" requires at least 1 argument(s).
See 'docker stop --help'.

Usage:  docker stop [OPTIONS] CONTAINER [CONTAINER...]

Stop one or more running containers
"docker rm" requires at least 1 argument(s).
See 'docker rm --help'.

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers
Makefile:30: recipe for target 'deploy' failed
make: *** [deploy] Error 123
```

**Comment by dmitrinesterenko** [comment]
In the /Makefile

**Comment by dmitrinesterenko** [comment]
- [ ] Deploy fails while container is running because the name is already taken.
- [ ] Is there a rotation strategy?

**Comment by marcus-campos** [comment]
has one solution for this bug?

**Comment by dmitrinesterenko** [comment]
@marcus-campos I've solved it with a somewhat hacky approach that brings down my application, this one can afford to have downtime, don't do this in an application that can not afford it. This was fixed up in #92 in these lines https://github.com/dmitrinesterenko/blog/blob/69a88003161425d9e32a9fb1995a7bfc53a4dc22/scripts/deployment/deploy.sh#L16-L18
```
#stop and rm the running container
docker stop blog
docker rm blog
#now start up the new one
docker run -d -p 80\:4000 -e "PORT=4000" -e "MIX_ENV=prod" --link blogphoenix_db_1:db --name blog dmitrinesterenko/blog\:latest
```

Are you trying to solve a similar issue and have a PR or comment to share?
